process/transaction/shardProcess.go

```diff
@@ -568,7 +568,7 @@ func (txProc *txProcessor) removeValueAndConsumedFeeFromUser(
 	if err != nil {
 		return err
 	}
-	consumedFee := txProc.economicsFee.ComputeFee(userTx)
+	consumedFee := core.SafeMul(userTx.GasLimit, userTx.GasPrice)
```

Comment:
So, this will take all the gas set in userTx for relayed txs? Could be a situation when GasLimit is set to 0 and we will not take anything?

Reply:
No. there is no way to have that. This is the correct way, until now that computation did only movebalance transfer cost which was always smaller than equal to the actual one.

Reply:
ok